### ISSUE TITLE
Fullscreen button looks not translated in french

### ISSUE THREAD

**Issue by rianfloo** [ableplayer/ableplayer#472]
Dear ableplayer team. 

I've setup ableplayer on a multi-language website. All translation work perfectly excepted one. 

The translation is related to the fullscreen icon which is not translated in FR. I had a look to the `translations/fr.js` file and was not able to find the translated variable. I found the variable 'Full Screen' but it is not linked to the full screen icon.

`<span class="able-clipped">Fullscreen</span>`

Do you have any solution?

Many thanks

**Comment by rianfloo** [comment]
Any news? :)

**Comment by terrill** [comment]
@rianfloo good catch. That's actually a bug in all languages, not just French. However, it shouldn't have any effect. The `<span>` you quoted is wrapped in a `<div>` with  `role="button"` and (in the French version) `aria-label="Activer le mode plein écran"`. The latter is what is read by screen readers, and it's also what appears visibly as a tooltip, so it's discoverable to speech recognition users.  

How did you discover this?  Did you encounter a use case that proves what I just said to be incorrect? 

In the short term I'll fix the bug that resulted in the content of that one `<span>` not being translated. But longer term, I'll probably remove the `<span>` because I don't think it's needed anymore.

**Comment by rianfloo** [comment]
Hi @terrill, I discovered this because I am using Ableplayer on a website aiming to get RGAA accessibility label (https://www.numerique.gouv.fr/publications/rgaa-accessibilite/) given by French Government. To get that label you need to pass an audit which failed on my client website because of few things including **that missing translation**. I really need to fix that issue but I don't want to fix it directly inside your source code. 

A new audit is planned next week and it would be wonderful if you can update this before. 👍  🍾 

Thanks for your amazing library! 🥇

**Comment by terrill** [comment]
@rianfloo any changes I make will be in the develop branch, and won't be merged into the main branch for a few more weeks (there are lots of other changes in the works, which ultimately will be released as version 4.4).  If you're ok using the latest version from the develop branch,  there's a lot of new text in [translations/fr.js](https://github.com/ableplayer/ableplayer/blob/develop/translations/fr.js) related to new features in the Audio Description Preferences dialog. See Issue #453. It would be great if you could contribute French translations for the new text! (and any other languages you're comfortable with)

**Comment by terrill** [comment]
The issue with the "Fullscreen" text should be fixed now, as of 679bb60a2f0c3bab537c5c82ec4bf46cb017bedf. 

Note that this will remain on the **develop** branch for a while; the new audio description settings require more testing before we can merge **develop** into **main** and release a new official version.  Other than the new audio description settings, all other changes are believed to be stable, so If you aren't using text-based audio description, you're probably safe to deploy the latest version from the **develop** branch.
